pkg/codegen/internal/test/testdata/simple-yaml-schema/schema.yaml

```
@@ -0,0 +1,126 @@
+version: "0.0.1"
+name: example
+types:
+  example::Object:
+    properties:
+      foo:
+        "$ref": "#/resources/example::Resource"
+      bar:
+        type: string
+      others:
+        type: array
+        items:
+          type: array
+          items:
+            "$ref": "#/types/example::SomeOtherObject"
+        description: List of lists of other objects
+      configs:
+        type: array
+        items:
+          "$ref": "#/types/example::ConfigMap"
+      stillOthers:
+        type: object
+        additionalProperties:
+          type: array
+          items:
+            "$ref": "#/types/example::SomeOtherObject"
+        description: Mapping from string to list of some other object
+    type: object
+  example::OtherResourceOutput:
+    properties:
+      foo:
+        type: string
+    type: object
+  example::SomeOtherObject:
+    properties:
+      baz:
+        type: string
+    type: object
+  example::ConfigMap:
+    properties:
+      config:
+        type: string
+    type: object
+  example::ObjectWithNodeOptionalInputs:
+    properties:
+      foo:
+        type: string
+      bar:
+        type: integer
+    required: [foo]
+    language:
+      nodejs:
+        "requiredInputs": []
```

Could this be unquoted? (Trying to see if there's a reason it's expressed with the quotes)
```suggestion
        requiredInputs: []
```

---

Same answer as above re: canonical representation of language-specific data.

---

In that case, shouldn't it be:
```
nodejs: {
	"requiredInputs": []
}
```